lblod/ember-rdfa-editor
# Embroider optimized support

### Overview
This should fix the remaining issues in #945 and make the addon fully Embroider "optimized" compatible.

##### connected issues and PRs:


Closes #945

### Setup



### How to test/reproduce
Run the embroider optimized scenario: `npx ember-cli try:one embroider-optimized` and the build should pass and the tests should succeed.

If you want to run the dummy app you could run `npx ember-cli try:one embroider-optimized --skip-cleanup` instead and run `EMBROIDER_TEST_SETUP_OPTIONS=optimized npm start` afterwards.

### Challenges/uncertainties




### Checks PR readiness
